addons/supplies/CfgWeapons.hpp

```diff
@@ -0,0 +1,12 @@
+class CfgWeapons {
+    class CBA_MiscItem;
+    class CBA_MiscItem_ItemInfo;
+    class GVAR(replacementBox): CBA_MiscItem {
```

Comment:
It might be better to name this more specific to the random chance mechanic.

Reply:
..Pick a name, let me know.

Reply:
`GVAR(randomizedBox)` :P